wrappers/javascript/indy-vdr-react-native/cpp/turboModuleUtility.cpp

```diff
@@ -143,7 +143,7 @@ int64_t jsiToValue(jsi::Runtime &rt, jsi::Object &options, const char *name,
                    bool optional) {
   jsi::Value value = options.getProperty(rt, name);
   if ((value.isNull() || value.isUndefined()) && optional)
-    return 0;
+    return -1;
```

Comment:
Is it correct to return -1 always in this case? 

In that case the above code that transforms it to -1 if it's `0` can also be removed

Reply:
Yes, after a quick check again -1 is interpreted as `None`. 

@wadeking98 could you fix all the `0`s to `-1`?

Reply:
@berendsliedrecht I've done it for int64 and int32 but I'm not sure what to do for uint 64,32,8 because it needs to return unsigned